docs/guides/expensive_notebooks.md

```diff
@@ -74,114 +74,40 @@ Learn more about exporting notebooks in our [exporting guide](../guides/exportin
 
 ## Caching
 
-marimo provides two caching utilities to help you manage expensive computations:
```

Comment:
Good cleanup. We do provide 3 "methods" with `lru`

I think it's worth highlighting the context block here opposed to cache vs persistent cache though


---
/// tab | `cache decorator`
```python
import marimo as mo
@mo.cache
def compute_embedding(data: str, embedding_dimension: int, model: str) -> np.ndarray:
    print("Skipped on cache hit")
    ...
```
///

/// tab | `cache block`

```python
import marimo as mo
with mo.persistent_cache("compute_embedding"):
    print("Skipped on cache hit")
    ...
```
///